spec/Candid.md

```diff
@@ -1247,6 +1247,9 @@ M(ref(r) : principal) = i8(0)
 M(id(v*) : principal) = i8(1) M(v* : vec nat8)
 ```
 
+Note:
+
+* Since `null` and `reserved` values take no space, to prevent unbounded sized message, we limit the length of `vec null` and `vec reserved` to be less than 2MiB.
```

Comment:
```suggestion
* Since `null` and `reserved` values, and records of such values, take no space, to prevent unbounded sized message, we limit the length of `vec null` and `vec reserved` to be less than 2MiB.
```